tests/integration/20.LB.Tests.ps1

```diff
@@ -77,6 +77,15 @@ Describe "Edge Load Balancer" {
             $lb_pool.member[1].name | Should be "VM02"
         }
 
+        it "Configure LB Pool" {
+            #Configure Pool to LB
+            Get-NsxEdge $lbedge1name | Get-NsxLoadBalancer | Get-NsxLoadBalancerPool -name pester_lb_pool1 | Set-NsxLoadBalancerPool -name pester_lb_pool3 -Description "Pester LB Pool 3" -Transparent:$true -Algorithm ip-hash
+            $lb_pool = Get-NsxEdge $lbedge1name | Get-NsxLoadBalancer | Get-NsxLoadBalancerPool pester_lb_pool3
```

Comment:
The tests failed due to line 112 where you only clean up 2 of the 3 lb pools. You need to make each test be independent of each other.

Maybe need to look to leverage the beforeeach and aftereach code blocks to create/cleanup the environment before/after each test.